database/model.go

```diff
@@ -476,7 +473,8 @@ func (db *Database) SelectTx(tx *sqlx.Tx, target interface{}, query *qb.SelectQu
 	if err != nil {
 		return errors.Wrap(err)
 	}
-	if err = db.DB.Select(target, stmt, values...); nil != err {
+
+	if err = tx.Select(target, stmt, values...); nil != err {
```

Comment:
:+1:

Reply:
I have been thinking we should create a wrapper for sqlx.Tx that includes the database, so we do not have to pass the two around together.